src/Http/RequestMatcher/FosRequestMatcher.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCacheBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCacheBundle\Http\RequestMatcher;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestMatcher as SymfonyRequestMatcher;
+
+/**
+ * Extend the Symfony RequestMatcher class to support query string matching.
+ */
+class FosRequestMatcher extends SymfonyRequestMatcher
```

Comment:
sorry to only come with this now, but can we please rename this to QuerystringRequestMatcher? `Fos` is not giving any hint what it does...